client/src/components/TeamTableContainer.tsx

```diff
@@ -0,0 +1,167 @@
+import { Team, User } from "../../lib/types";
+import TeamTable from "./TeamTable/TeamTable";
+
+export interface TeamWithEmployeesManager {
+  team: Team;
+  employees: User[];
+  manager: User;
+}
```

Comment:
If you're using this in diff files, add it in types, only in component when using it in the same component